spec/unit/jenkins_plugins_spec.rb

```diff
@@ -137,20 +137,34 @@
 
   describe '.plugins_from_updatecenter' do
     subject(:plugins) { described_class.plugins_from_updatecenter(fixture) }
-    let(:fixture) { File.expand_path(File.dirname(__FILE__) + '/../fixtures/update-center.json') }
 
-    it { should be_instance_of Hash }
-    # Our fixture file currently has 870 plugins in it
-    its(:size) { should eql 870 }
+    if RUBY_VERSION == '1.9.3'
```

Comment:
It would be nice to add a comment here about why this was done.  Over time, re-factoring often makes it hard to track don't commit messages with just `git blame`.